library/gen_x12_837.inc.php

```diff
@@ -440,11 +367,12 @@ function gen_x12_837($pid, $encounter, &$log, $encounter_claim=false) {
   //   $log .= "*** CMS ID is missing for payer '" . $claim->payerName() . "'.\n";
   // }
 
-  if (true) { // !$CMS_5010
+  if (true) { 
```

Comment:
As it does nothing at all?  Well shouldn't we keep a little useless legacy code in?  Ain't that tradition?
Making changes now.